spring-test/common/src/main/java/io/camunda/zeebe/spring/test/SimpleMetricsRecorder.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2021 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.camunda.zeebe.spring.test;
+
+import io.camunda.client.metrics.MetricsRecorder;
+import java.util.HashMap;
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicLong;
+
+/** Super simple class to record metrics in memory. Typically used for test cases */
+public class SimpleMetricsRecorder implements MetricsRecorder {
```

Comment:
@jonathanlukas in that regard the upstream change is actually breaking, do you consider it likely any user actually made use of it? 🤔 was the only purpose seemed to have been testing/ZPT or were there other use cases?

Reply:
Not really. This MetricsRecorder has no use case other than recording metrics and accessing them later in a test. Also, this was never publicly documented.

To be 100% sure, we could add this to CPT as well. @saig0 wdyt?